### ISSUE TITLE
I hate asking this ... but... 500 error.

### ISSUE THREAD

**Issue by CyberSecGuy** [Cyberbyte-Studios/CyberWorks#25]
Things I have done:

1) sudo a2enmod rewrite
2) .htaccess exists and is owned by apache; it is as follows.

```
RewriteEngine On
RewriteBase /
RewriteCond %{REQUEST_FILENAME} !-f
RewriteRule . /index.php [L]

php_value file_get_contents 1
```

**Comment by MightySCollins** [comment]
Anything in the logs?

**Comment by Provokan** [comment]
Which Error u become on the display?

**Comment by MightySCollins** [comment]
Also when you say apache what OS are you running on? Because if I remember correctly it should be `www-data` for debian. Also be sure to allow overrides in the vHosts file

**Comment by CyberSecGuy** [comment]
Even with editing: /etc/php5/apache2/php.ini, i'm not getting any error's to display, nor in the logs.
- Ubuntu 15.10
- PHP 5.6.11-1ubuntu3.1 (cli)
- Server version: Apache/2.4.12 (Ubuntu)

and yes, everything is www-data:www-data

**Comment by Provokan** [comment]
please, give me the url to your homepage (:

**Comment by CyberSecGuy** [comment]
Issue resolved - I don't know specifically what fixed it, I switched from Ubuntu 15.04 to Debian 8, and now it works.
